test/api/request/request.test.js

```diff
@@ -16,6 +16,10 @@ describe('#Request', () => {
   let nodeEnv;
 
   beforeEach(() => {
+    global.kuzzle = {
+      id: 'nasty-author-4242'
+    };
```

Comment:
With #1913, you'll have to delete `global.kuzzle` first, with `Reflect.deleteProperty`, otherwise this line will silently fail, and your tests won't behave correctly.